pyracf/resource/resource_admin.py

```diff
@@ -589,3 +718,43 @@ def _format_profile(self, result: dict) -> None:
 
         del result["securityResult"]["resource"]["commands"][0]["messages"]
         result["securityResult"]["resource"]["commands"][0]["profiles"] = profiles
+
+    def __validate_access_level(
+        self,
+        success: Union[str, None] = None,
+        failure: Union[str, None] = None,
+        all: Union[str, None] = None,
+    ):
+        access_levels = ["alter", "control", "read", "update"]
+        value_error_text = (
+            "Please specify 'alter', 'control', 'read', or 'update' access "
+            + "for your target attempt."
+        )
+        bad_values = []
+        if success is not None:
+            if success.lower() not in access_levels:
+                bad_values.append(success)
+        if failure is not None:
+            if failure.lower() not in access_levels:
+                bad_values.append(failure)
+        if all is not None:
+            if all.lower() not in access_levels:
+                bad_values.append(all)
+        if len(bad_values) == 0:
+            return
+        elif len(bad_values) == 1:
+            value_error_text = (
+                f"'{bad_values[0]}' is not a proper value. " + value_error_text
```

Comment:
```python
f"'{bad_values[0]}' is not a valid access level. " + value_error_text
```

Similar updates should be made to the other error text conditions as well.

Reply:
I wanted to stay away from calling these access levels in messaging even though they are, because this is for the `by_attempt` functions. I think it will be confusing to have people see those terms intermingling even though its true. The `by_attempt` functions you use the attempt as the argument and the value you pass in is an access level and for the `by_access_level` functions the access level is the argument and you pass in an attempt type.

Reply:
I disagree. The `by_atempt` functions take three key-value pairs in the format `attempt=access_level`. Saying that the argument provided `is not a valid access level.` is more specific than `is not a proper value`. It also has the added benefit of being an educational aid since it teaches the user that the things that they are specifying as parameters are access levels.

Reply:
sounds good, you have convinced me.